TypeCobol/AntlrGrammar/CobolWords.g4

```diff
@@ -879,7 +879,8 @@ tokens
 	SQL_XMLCAST,
 	SQL_YEAR,
 	SQL_YEARS,
-	SQL_ZONE
+	SQL_ZONE,
+    SQL_CommaSeparator
```

Comment:
Case is fine but indent is not, could you check that ?

Reply:
By default Visual Studio replaces tabs with spaces so use Notepad to ensure the indent is correct.